index.html

```diff
@@ -104,7 +104,5 @@ <h5 class="modal-title" id="errorModalLabel">Error</h5>
   <link rel="stylesheet" href="https://code.jquery.com/qunit/qunit-2.19.1.css">
   <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
   <script src="https://code.jquery.com/qunit/qunit-2.19.1.js"></script>
```

Comment:
Hang on, none of the scrips/css etc should be in the files for the actual pages. They should be in _layouts/base.html. Every page uses this.

For test stuff, perhaps we can have a dedicated template that builds on the base so that the unit test stuff is only loaded when needed?